haiku-games/bevexed/bevexed-20141224.recipe

```diff
@@ -48,4 +47,4 @@ INSTALL()
 
 	cp -a source/objects/BeVexed $appsDir
 	addAppDeskbarSymlink $appsDir/BeVexed
-}
+}
```

Comment:
This line is missing a linefeed. If you edit the file with vi and save it then vi will fix that for you.